lib/IDE/TypeReconstruction.cpp

```diff
@@ -1186,6 +1186,37 @@ static void VisitNodeFunction(
       VisitNode(ast, *pos, decl_scope_result);
       break;
 
+    case Demangle::Node::Kind::PrivateDeclName: {
+      if (child->getNumChildren() != 2 || !child->getChild(0)->hasText() ||
```

Comment:
Can you combine this code with the existing function VisitNodePrivateDeclName()?

Reply:
Done